server/src/main/java/org/opensearch/index/translog/RemoteFsTranslog.java

```diff
@@ -49,6 +49,8 @@ public class RemoteFsTranslog extends Translog {
 
     private volatile long minSeqNoToKeep;
 
+    private volatile long minRemoteGenReferenced;
```

Comment:
let's add java doc for this new variable.

Reply:
done.